projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -130,8 +130,11 @@ public function add_users_menu() {
 
 		add_submenu_page( 'users.php', esc_attr__( 'Subscribers', 'jetpack' ), __( 'Subscribers', 'jetpack' ), 'list_users', 'https://wordpress.com/subscribers/' . $this->domain, null );
 
-		remove_submenu_page( 'users.php', 'profile.php' );
-		add_submenu_page( 'users.php', esc_attr__( 'My Profile', 'jetpack' ), __( 'My Profile', 'jetpack' ), 'read', 'https://wordpress.com/me/', null );
+		// When the interface is set to wp-admin, we replace the Profile submenu.
```

Comment:
I think this comment is not accurate, it should say something like "when the interface is NOT wp-admin" ?